viz/charts/linkerd-viz/Chart.yaml

```
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-viz"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.12.2-edge
+version: 30.12.3-edge
```

We made a mistake in `edge-23.3.3` in that we bumped the patch instead of the minor and so ended up with:
```
edge-23.9.3: 30.12.2-edge
stable-2.14.1: 30.12.2
```
and so in semver terms `edge-23.9.3` < `stable-2.14.1` when it should have been the other way around.
Nothing that we can do now to fix that, but probably not a big deal anyways. 
But to still have `edge-23.9.4` to come after `stable-2.14.2` and any 2.14 maintenance release that comes after, we should have here:

```suggestion
version: 30.13.0-edge
```
This only happened for the viz and multicluster charts. The others are fine.

---

Turns out linkerd2-cni also has this problem, so I'd say we set that chart's version to `30.13.0-edge` as well, in this PR to get that right from now own, even if that chart didn't receive any modifications since the last edge.